src/java_bytecode/generate_java_generic_type.cpp

```diff
@@ -11,23 +11,10 @@
 
 #include "generate_java_generic_type.h"
 #include <util/namespace.h>
+#include <util/prefix.h>
```

Comment:
One more tiny nit that just sprung to mind (sorry!) Now. that the function has been moved out of this file, probably this include is not needed anymore either...